### ISSUE TITLE
CO2.0: Error when clicking though to related items

### ISSUE THREAD

**Issue by jamieu** [TheScienceMuseum/collectionsonline#1538]
This may be similar to the issue I saw last week which has since resolved itself, but I am now seeing errors when clicking though from related items. 

ie. http://localhost:8000/objects/co8227672/pair-of-candlesticks-sheffield-plate-by-boulton-and-fothergill-birmingham-candlestick then clicking on 'crumpled paper' bottom right.

I think this may be related to the 'child records' redirecting to the 'parent record':

For example co8217456  (box-containing-crumpled-paper / a child record) is a part of co8217456 (james-watts-garret-workshop / a parent SPH record)
http://localbhost:8000/objects/co8217456/box-containing-crumpled-paper-and-red-powder-labelled-best-lake-box

But when you follow that link (right-click one in new tab) for co8217456 you get taken to co52335 (the parent record / workshop).
http://localhost:8000/objects/objects/co52335/james-watts-garret-workshop-used-1790-1819-workshop-work-space

Which itself seems to throw.an error?
localhost:8000/objects/co52335/james-watts-garret-workshop-used-1790-1819-workshop-work-space

```
Service Unavailable
unavailable

Error: unavailable
    at handler (/Users/jamieunwin/projects/collectionsonline/routes/object.js:60:23)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async exports.Manager.execute (/Users/jamieunwin/projects/collectionsonline/node_modules/@hapi/hapi/lib/toolkit.js:60:28)
    at async internals.handler (/Users/jamieunwin/projects/collectionsonline/node_modules/@hapi/hapi/lib/handler.js:46:20)
    at async exports.execute (/Users/jamieunwin/projects/collectionsonline/node_modules/@hapi/hapi/lib/handler.js:31:20)
    at async Request._lifecycle (/Users/jamieunwin/projects/collectionsonline/node_modules/@hapi/hapi/lib/request.js:370:32)
    at async Request._execute (/Users/jamieunwin/projects/collectionsonline/node_modules/@hapi/hapi/lib/request.js:280:9)
<img width="1218" alt="Screenshot 2024-02-05 at 14 37 55" src="https://github.com/TheScienceMuseum/collectionsonline/assets/91365/e896d983-3386-4ce3-ab95-da4759079179">
```

**Comment by jamieu** [comment]
This seems to have been fixed, awaiting testing

One possibility of why this error was resolved at the same times as we fixed the filters/weighting for related objects, is that previously it threw an error in the following (now rare) scenarios.

-  The related  objects have no images
-  The record has no related objects ie. a category of one. 

Now that the search weights are fixed, the above are very rare occurrences and hard to test for. But we should keep in mind this issue if we spot this kind of behaviour in future. We should also look out for examples of the above to test with in future.
